examples/cpp/embed-cl/train_embed-cl.cc

```diff
@@ -156,7 +156,7 @@ int main(int argc, char** argv) {
     for (unsigned i = 0; i < report_every_i; ++i) {
       if (si == training.size()) {
         si = 0;
-        if (first) { first = false; } else { sgd->update_epoch(); }
```

Comment:
Thanks, this is great. We can also probably remove the "first" variable entirely, right?

Reply:
Done. Please take a look.